fabricadesoftware-ifc/fabricaNode-BackEnd
# FIX: Duplicate Area and Subarea

### FIX DUPLICATION ON AREAS AND SUBAREAS NAME

**Descripition:**

This MR involves fixing a bug that allowed duplicating the names of areas and subareas

**Test:**

1. Access the endpoint: "/api/areas/"
2. Try register two areas with the same name
3. Check if everything is ok
4. Access the endpoint: "/api/subareas/"
5. Try register two subareas with the same name
6. Check if everything is ok
